crates/swift-integration-tests/src/opaque_type_attributes/copy.rs

```diff
@@ -1,7 +1,7 @@
 #[swift_bridge::bridge]
 mod ffi {
     extern "Rust" {
-        #[swift_bridge(Copy(6))]
+        #[swift_bridge(Copy(6), Equatable)]
```

Comment:
Please create a separate type such as `RustCopyAndEquatableType` so that we can remain confident that `Copy` works without `Equatable`.